api/src/libs/healthCheck.ts

```diff
@@ -5,13 +5,13 @@ type Status = "PASS" | "FAIL" | "ERROR";
 type StatusResult = Record<string, Status>;
 
 const healthCheckEndPoints: Record<string, string> = {
-  self: "self2",
-  elevator: "dynamics/elevator",
-  fireSafety: "dynamics/fire-safety",
-  housing: "dynamics/housing",
-  dynamicsLicenseStatus: "dynamics/license-status",
-  webserviceLicenseStatus: "webservice/license-status",
-  webserviceFormation: "webservice/formation",
+  self: "self2Error",
```

Comment:
[pebble] This is not needed. `self2` was already the change that broke the self-check, since the self-check path is `self`, not `self2`.

Reply:
done